lnd.go

```diff
@@ -46,7 +47,25 @@ func (svc *LNDService) GetBalance(ctx context.Context, senderPubkey string) (bal
 }
 
 func (svc *LNDService) MakeInvoice(ctx context.Context, senderPubkey string, amount int64, description string, descriptionHash string, expiry int64) (invoice string, paymentHash string, err error) {
-	return "", "", fmt.Errorf("not implemented")
+	var descriptionHashBytes []byte
+	
+	if descriptionHash != "" {
+		descriptionHashBytes, err = hex.DecodeString(descriptionHash)
+		if err == nil && len(descriptionHashBytes) != 32 {
+			err = errors.New("Description hash must be 32 bytes hex")
+		}
+		if err != nil {
```

Comment:
`err == nil` is considered bad practice. I would combine the 2 cases here: `if err != nil || len != 32 { return new error}`